lib/node_modules/@stdlib/math/base/special/sqrtpi/package.json

```
@@ -0,0 +1,69 @@
+{
+  "name": "@stdlib/math/base/special/sqrtpi",
+  "version": "0.0.0",
+  "description": "Compute the positive square root of the product of pi and a provided positive number.",
+  "license": "Apache-2.0",
+  "author": {
+    "name": "The Stdlib Authors",
+    "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+  },
+  "contributors": [
+    {
+      "name": "The Stdlib Authors",
+      "url": "https://github.com/stdlib-js/stdlib/graphs/contributors"
+    }
+  ],
+  "main": "./lib",
+  "gypfile": true,
+  "directories": {
```

```suggestion
  "directories": {
```

No native implementation, and thus no `binding.gyp` file is present in the package.